lib/deaf_grandma.rb

```diff
@@ -14,14 +14,26 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      exit if @bye_counter == 3
```

Comment:
Since `@bye_counter` is compared with `3` in two places, consider a new instance method called `exiting?`.